_posts/2022-12-09-trino-summit-2022-shopify-recap.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Rewriting History: Migrating petabytes of data to Apache Iceberg using Trino"
+author: "Marc Laforet, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/shopify.jpg
+---
+
+Rolling right along with another one of [our Trino Summit 2022 recap posts]({% post_url
+2022-11-21-trino-summit-2022-recap %}), we're excited to bring you the engaging
+talk from Marc Laforet at Shopify. He talked about the ordeal (or, if you look
+at it in a positive light, the privilege) of migrating petabytes of data from
+Hive to Iceberg table formats with the help of Trino. With details on why
+Shopify chose to move to Iceberg, the various migration strategies that were
+considered, and the ultimate process of moving all that data while the Trino
+Iceberg connector was still in active development, it's an insightful talk that
```

Comment:
```suggestion
Iceberg connector was still in active development. This is an insightful talk that
```

Reply:
This would make the first half of the split a sentence fragment.